src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1985,15 +1986,15 @@
     <Kind Name="OrPattern"/>
     <Kind Name="AndPattern"/>
     <Field Name="LeftPattern" Type="PatternSyntax" />
-    <Field Name="PatternOperator" Type="SyntaxToken">
+    <Field Name="OperatorToken" Type="SyntaxToken">
       <Kind Name="OrKeyword"/>
       <Kind Name="AndKeyword"/>
     </Field>
     <Field Name="RightPattern" Type="PatternSyntax" />
   </Node>
   <Node Name="UnaryPatternSyntax" Base="PatternSyntax">
     <Kind Name="NotPattern"/>
-    <Field Name="PatternOperator" Type="SyntaxToken">
+    <Field Name="OperatorToken" Type="SyntaxToken">
       <Kind Name="NotKeyword"/>
     </Field>
     <Field Name="Pattern" Type="PatternSyntax" />
```

Comment:
>Pattern [](start = 17, length = 7)

"Operand" for consistency with unary expressions?

Reply:
I don't think so.  The analogy with expressions breaks down here.  The unary pattern is not an operation, so it doesn't really have an operand.

---
In reply to: [412377963](https://github.com/dotnet/roslyn/pull/43526#discussion_r412377963) [](ancestors = 412377963)

Reply:
i'm ok with these being different as well.  it was the binary ones that felt super weird.